src/Common/src/Interop/Linux/libc/Interop.sockaddr.cs

```diff
@@ -16,6 +16,7 @@ internal static partial class libc
         public const sa_family_t AF_UNIX = 1;   // Local to host.
         public const sa_family_t AF_INET = 2;   // IP protocol family.
         public const sa_family_t AF_INET6 = 10; // IP version 6.
+        public const sa_family_t AF_PACKET = 17; // Packet family
```

Comment:
The changes to this file are now unused, all of this has been moved to the shim.